flash/image/face_detection/backbones/fastface_backbones.py

```diff
@@ -0,0 +1,43 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from functools import partial
+
+from flash.core.registry import FlashRegistry
+from flash.core.utilities.imports import _FASTFACE_AVAILABLE
+
+if _FASTFACE_AVAILABLE:
+    import fastface as ff
+
+
+model_name = ["lffd_slim", "lffd_original"]
```

Comment:
```suggestion
_MODEL_NAMES = ["lffd_slim", "lffd_original"]
```

Reply:
It would be better if we do it like this I guess;
```python
_MODEL_NAMES = ff.list_pretrained_models()
```
There are new models on the road